opencti-platform/opencti-front/tests_e2e/model/menu/leftBar.pageModel.ts

```diff
@@ -0,0 +1,17 @@
+import { Page } from "@playwright/test";
+import { expect } from '../../fixtures/baseFixtures';
+
+export class LeftBarPage {
+  constructor(private page:Page) {
+  }
+
+  async clickOnMenu(menuName: string, pageName: string) {
+    await this.page.getByRole('menuitem', { name: menuName }).click();
+    await this.page.getByRole('menuitem', { name: pageName }).click();
+  }
+
+  async checkPage(menuName: string, pageName: string) {
```

Comment:
```suggestion
  async expectPage(menuName: string, pageName: string) {
```
IMO keep the same wording `expect` as the test lib  when we want custom assertion functions

Reply:
Done